geode-core/src/main/java/org/apache/geode/internal/lang/SystemPropertyHelper.java

```diff
@@ -70,12 +65,9 @@
    * @param name system property name set in Geode
    * @return a boolean value of the system property
```

Comment:
Should these javadocs be updated since the method no longer returns a boolean value?

Reply:
Yes, changed.